src/parser/monk/shared/modules/covenants/FallenOrder.tsx

```diff
@@ -0,0 +1,144 @@
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+import SPELLS from 'common/SPELLS';
+import Events, { DamageEvent, HealEvent, SummonEvent } from 'parser/core/Events';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import ItemHealingDone from 'interface/ItemHealingDone';
+import React from 'react';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import { formatNumber } from 'common/format';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+
+class FallenOrder extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+    abilities: Abilities,
+  };
+
+  windDamage: number = 0;
+  brewDamage: number = 0;
+  mistDamage: number = 0;
+
+  mistHealing: number = 0;
+  mistOverhealing: number = 0;
+  cloneMap: Map<number, number> = new Map<number, number>();
+
+
+  constructor(options: any) {
+    super(options);
+
+    // In my testing this line always returns false but its what putput uses so I have faith
+    // it will work in the future. (might just be a bad log too)
+    this.active = this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
+
+    if (!this.active) {
+      return;
+    }
+
+    options.abilities.add({
+      spell: SPELLS.FALLEN_ORDER_CAST,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      cooldown: 180,
+      gcd: {
+        base: 1500,
+      },
+      castEfficiency: {
+        suggestion: true,
+        recommendedEfficiency: 0.8,
+      },
+    });
+
+    //summon events (need to track this to get melees)
+    this.addEventListener(Events.summon.by(SELECTED_PLAYER).spell([SPELLS.FALLEN_ORDER_OX_CLONE, SPELLS.FALLEN_ORDER_TIGER_CLONE, SPELLS.FALLEN_ORDER_CRANE_CLONE]), this.trackSummons);
+
+    //mistweaver spells
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER_PET).spell([SPELLS.FALLEN_ORDER_ENVELOPING_MIST, SPELLS.FALLEN_ORDER_SOOTHING_MIST]), this.mistHealingTracker);
+    //brewmaster spells
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER_PET).spell([SPELLS.FALLEN_ORDER_KEG_SMASH, SPELLS.FALLEN_ORDER_BREATH_OF_FIRE,SPELLS.BREATH_OF_FIRE_DEBUFF]), this.brewDamageTracker);
+    //windwalker spells
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER_PET).spell([SPELLS.FALLEN_ORDER_SPINNING_CRANE_KICK, SPELLS.FISTS_OF_FURY_DAMAGE]), this.windDamageTracker);
```

Comment:
Is it FISTS_OF_FURY for both WW and other specs? No special ID for FO?

Reply:
Fallen Order clones are weird af where sometimes it uses the core spell and sometimes they recreate it. In this instance they use the core spell for FoF but then recreate SCK